src/Repository/ProductRepository.php

```diff
@@ -31,6 +31,20 @@ public function getAllActiveProducts($onlyId = false)
         return $this->db->getValue($query);
     }
 
+    public function getAllActiveDpdProductReferences($onlyId = false)
```

Comment:
Might as well create two different function, one for product information one for only id_reference.

Reply:
Seems like a new function, could still split it into two actually usable functions with SOLID principles